backend/internal/supabase/migrations/20241127192734_notifications.sql

```diff
@@ -0,0 +1,12 @@
+create type notif_type as enum ('follow', 'comment', 'upvote');
+
+CREATE TABLE notifications (
+    id SERIAL PRIMARY KEY,
+    type notif_type NOT NULL,
+    created_at timestamp with time zone default now(),
+    reciever_id uuid NOT NULL references "user"(id),    
```

Comment:
```suggestion
    receiver_id uuid NOT NULL references "user"(id),    
```
make sure to update spelling in SQL queries too